src/main.rs

```diff
@@ -13,6 +13,7 @@ const CAREFUL_FLAGS: &[&str] = &[
     "-Zstrict-init-checks",
     "-Zextra-const-ub-checks",
     "-Cdebug-assertions=on",
+    "-Coverflow-checks=on",
```

Comment:
We already set  debug-assertions=on, so I think this is a NOP?

Reply:
If e.g. `profile.dev` set overflow checks off, would this still be a NOP?

Reply:
For the sysroot build, yes. Crate profiles are not relevant. Though I guess if you set this as global profile it would still apply to the sysroot build.

For the local crate, not sure. If you set that in the profile should we really overwrite it?

Reply:
You're right, TIL: https://github.com/rust-lang/rust/blob/master/compiler/rustc_session/src/session.rs#L1056

I had always thought these were separate and only related by the fact that the `dev` profile sets both. Good to know.